test/resolution/resolver.test.ts

```diff
@@ -1358,6 +1358,15 @@ describe("Resolve", () => {
       expect(subject1 === subject2).eql(true);
   });
 
+  it("should not deactivate a non activated constant value", () => {
+    const container = new Container();
+    let deactivated = false;
+    container.bind<string>("ConstantValue").toConstantValue("Constant").onDeactivation(_ => {deactivated = true;})
+    container.unbind("ConstantValue");
+    expect(deactivated).to.equal(false);
+  });
```

Comment:
I'd suggest using sinnon spies next time, but we can afford not to do it now

Reply:
but then would have had got access to the binding.

Reply:
Sorry I don't understand your point :( (my english is far to be perfect)

Reply:
> Sorry I don't understand your point :( (my english is far to be perfect)

There was no point to my point !

You are correct, I should have done
```

  it("should not deactivate a non activated constant value", () => {
    const container = new Container();
    const deactivationHandler = sinon.spy();
    container.bind<string>("ConstantValue").toConstantValue("Constant").onDeactivation(deactivationHandler)
    container.unbind("ConstantValue");
    // tslint:disable-next-line: no-unused-expression
    expect(deactivationHandler.notCalled).to.be.true;
  });
```

Will change

Reply:
Done